exercises/edward_brown/ex01_basics/src/ToLowerCase.cpp

```diff
@@ -0,0 +1,10 @@
+#include "ToLowerCase.h"
+#include <algorithm>
+#include <cctype>
+
+std::string toLowerCase(std::string MixedOrUpperCase) {
+  std::transform(MixedOrUpperCase.cbegin(), MixedOrUpperCase.cend(),
+                 MixedOrUpperCase.begin(),
+                 [](char C) -> char { return std::tolower(C); });
```

Comment:
Good spot! But then shouldn't the argument to the lambda function be `unsigned char`?